### PR TITLE
Fix return type validation

### PR DESCRIPTION
**Pull request by jeromelaban**
Fix for https://github.com/unoplatform/uno/issues/2239

### CODE REVIEW

**Review by davidjohnoliver — APPROVED** [review]

**Review by carldebilly — APPROVED** [review]

File: .vsts-ci.yml [hunk 1]
```diff
@@ -71,9 +71,7 @@ jobs:
     clean: all
 
   pool:
-    name: Default
-    demands:
-    - agent.os -equals Linux
+    vmImage: 'ubuntu-18.04'
```
Comment (dr1rrb) [inline .vsts-ci.yml:74]: seems unrelated to this PR

**Review by dr1rrb — APPROVED** [review]

File: .vsts-ci.yml (hunk 1 above)
Comment (jeromelaban) [reply .vsts-ci.yml:74]: It is :) ADO changed build pipelines
